public/index.html

```diff
@@ -24,7 +24,7 @@ <h1>
                     Welcome to Go Outside Co.
                     <br>Equipment Reservations
                 </h1>
-                <nav>reservations home || <a href="login.html">login</a></nav>
+                <nav>reservations home || <a href="login.html" id="log">log in</a></nav>
```

Comment:
log is a common name would `loginLink` be a better id?

Reply:
changed all instances of id='log' to id='user_status'